verify/build-secrets

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -euf
+
+BASE=${1:-./secrets}
+tar -czvf cockpit-verify-secrets.tgz -C "$(dirname $BASE)" "$(basename $BASE)"
```

Comment:
Hmm. Is that any simpler than scp'ing the `secrets/` directory? If you want/need it, maybe at least chown it to 600?